tests/installation/bootloader_svirt.pm

```diff
@@ -212,7 +213,9 @@ sub run {
         my $cmdline = '';
         $cmdline .= 'textmode=1 ' if check_var('VIDEOMODE', 'text');
         $cmdline .= 'rescue=1 ' if is_installcheck || is_rescuesystem;    # rescue mode
-        $cmdline .= get_var('EXTRABOOTPARAMS') . ' ';
+        $cmdline .= get_var('EXTRABOOTPARAMS') . ' ' if get_var('EXTRABOOTPARAMS');
+        $cmdline .= registration_bootloader_cmdline if check_var('SCC_REGISTER', 'installation');
+
```

Comment:
I don't see the need for the extra blank line.

Reply:
Fixed